test/functional/page_objects/dashboard_page.ts

```diff
@@ -562,7 +562,7 @@ export class DashboardPageObject extends FtrService {
 
   public async getPanelTitles() {
     this.log.debug('in getPanelTitles');
-    const titleObjects = await this.testSubjects.findAll('dashboardPanelTitle');
+    const titleObjects = await this.find.allByCssSelector('span.embPanel__titleInner');
```

Comment:
When selecting the `dashboardPanelTitle` test subject and getting the inner text of the resulting element, this would cause the badges to be included in the title - for example, consider this panel: 

![image](https://user-images.githubusercontent.com/8698078/181077549-2373d053-e3d5-4f37-b955-91cc9144ab51.png)

The old code would result in title similar to `"Aggregation - Area\nLast 1 week\nCustom filters"` while the new code will result in simply `"Aggregation - Area"`